conda/install.py

```diff
@@ -86,6 +88,36 @@ def win_soft_link(src, dst):
         if not CreateSymbolicLink(dst, src, isdir(src)):
             raise OSError('win32 soft link failed')
 
+    def win_conda_bat_redirect(src, dst):
+        """Special function for Windows XP where the `CreateSymbolicLink`
+        function is not available.
+
+        Simply creates a `.bat` file at `dst` which calls `src` together with
+        all command line arguments.
+
+        Works of course only with callable files, e.g. `.bat` or `.exe` files.
+        """
+        try:
+            os.makedirs(os.path.dirname(dst))
+        except OSError as exc:  # Python >2.5
+            if exc.errno == errno.EEXIST and os.path.isdir(os.path.dirname(dst)):
+                pass
+            else:
+                raise
+
+        # bat file redirect
+        with open(dst+'.bat', 'w') as f:
+            f.write('@echo off\n"%s" %%*\n' % src)
+
+        # TODO: probably need one here for powershell at some point
+
+        # This one is for bash
+        if src.endswith("conda"):
+            src = src + ".exe"
+
+        p = subprocess.check_call(["bash", "-l", "-c", "ln -sf {src} {dst}".format(
```

Comment:
thanks.  I have had that in here, but removed it for testing so that I see any errors.  I have had a lot of trouble getting output from py.test - it seems the -s flag doesn't always work somehow.  I'll re-do the exception handling you suggest, as it is definitely the right way to do things.